src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int INDEX = 5;
+    static final int SIDE = 10;
+    private static final String DEFAULT_COLOR = String.valueOf(Color.WHITE);
```

Comment:
```suggestion
    private static final String DEFAULT_COLOR = Color.WHITE.name();
```
you should use `name()` to get String of a enum